src/styles/blocks/page.scss

```
@@ -0,0 +1,101 @@
+body {
+  @include main-font-set;
+
+  margin: 0;
+}
+
+.page-cont {
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  @include main-inline-padding;
+
+  width: 200px;
+  height: 406px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    background: url(../images/imac.jpeg) no-repeat center/contain;
+
+    width: 160px;
+    min-height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin: 0 0 4px;
```

reset default margin to use like this
```suggestion
    margin-bottom: 4px;
```